dts/bindings/dma/nordic,easydma-device.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Nordic nRF family EasyDMA
+
+  This is the binding for the peripherals with EasyDMA support.
```

Comment:
I agree with @anangl that a comment would be better than a description since this is an include file, not a binding for a compatible.

I don't see any `memory-region` property defined in `nordic,nrf-spim.yaml`, so I am confused about the second comment, though.

Reply:
> I don't see any `memory-region` property defined in `nordic,nrf-spim.yaml`, so I am confused about the second comment, though.

As I understand it, that's the purpose of the added file, to be included from bindings for nRF peripherals that use EasyDMA, so that this property can be defined in their devicetree nodes if needed. That's why I think this PR should also present the use of that file.

Reply:
Sure I can add the inclusion of this binding here, but initially, it was done here https://github.com/zephyrproject-rtos/zephyr/pull/45128 and I thought that after this PR is merged I'll align the one I mentioned. So @mbolivar-nordic and @anangl would you like to have it done here?

Reply:
>  So @mbolivar-nordic and @anangl would you like to have it done here?

Yes, I think that makes sense.